pkg/security/secl/model/model.go

```diff
@@ -131,6 +131,9 @@ type BaseEvent struct {
 	Service       string         `field:"event.service,handler:ResolveService,opts:skip_ad|gen_getters"` // SECLDoc[event.service] Definition:`Service associated with the event`
 	Hostname      string         `field:"event.hostname,handler:ResolveHostname"`                        // SECLDoc[event.hostname] Definition:`Hostname associated with the event`
 
+	// Fields populated upon rule evaluation
+	RuleTags []string `field:"rule.tags"` // SECLDoc[rule.tags] Definition:`Tags associated with the rule that's used to evaluate the event`
```

Comment:
should we keep the same syntax has the other event level field ? `event.rule.tags`